### PR TITLE
Bugfix/mar 802 non auth user

### PR DESCRIPTION
**Pull request by punkrussian72**
MAR-856 correct link to polkadot wiki

### CODE REVIEW

File: src/pages/MyTokens/NFTPage.tsx [hunk 1]
```diff
@@ -93,11 +94,16 @@ export const NFTPage = () => {
...
   const [isFetchingTokens, setIsFetchingTokens] = useState<boolean>(false);
   const deviceSize = useDeviceSize();
+  const navigate = useNavigate();
 
   const { offers, isFetching: isFetchingOffers, fetch } = useOffers();
 
   const { api } = useApi();
 
+  useEffect(() => {
+    if (!isLoading && !selectedAccount) navigate('/');
```
Comment (SergeyOkishev) [inline src/pages/MyTokens/NFTPage.tsx:104]: 1. it looks out of place (redirect from NFT page? Why)
2. Don't redirect any one, no need to. We disabled buttons, that's all, no force.

Comment (punkrussian72) [reply src/pages/MyTokens/NFTPage.tsx:104]: removed

File: src/account/AccountProvider.tsx [hunk 2]
```diff
@@ -146,11 +127,55 @@ const AccountWrapper: FC = ({ children }) => {
...
+      };
+
+      listenExtensionChanges();
+      return () => {
+        unsubscribe && unsubscribe();
+      };
+    }
+  }, [web3EnablePromise, fetchAccounts]);
+
+  const getExtensionAccounts = useCallback(async () => {
```
[the remark this answers is not in the document]
Comment (punkrussian72) [reply src/account/AccountProvider.tsx:152]: In the very beginning I subscribed to polkadot extension changes in the getExtensionAccounts method. And there were errors that I was using some methods before their declaration. 
Moved those methods back.

File: src/account/AccountProvider.tsx [hunk 3]
```diff
@@ -28,6 +29,12 @@ const AccountWrapper: FC = ({ children }) => {
     setSelectedAccount(account);
   }, []);
 
+  const clearSelectedAccount = useCallback(() => {
```
[the remark this answers is not in the document]
Comment (punkrussian72) [reply src/account/AccountProvider.tsx:32]: You are correct. I've renamed this function

**Review by SergeyOkishev — APPROVED** [review]

**Pull request merged**